contents/newsletter/small-teams.md

```diff
@@ -0,0 +1,96 @@
+
+Team leaders do not automatically equal managers. Team leads are responsible for making sure that teams perform well and for giving direct feedback. Managers have a different remit and are focused on the happiness and career progression of direct reports, onboarding new hires, and discussing performance issues with the executive team. A team lead’s remit is deliberately more product-focused.
+
+
+### They don’t clash
+
+Creating a startup full of startups sounds like a recipe for people duplicating work or stepping on toes, right? Wrong. If you combine a small team structure with a transparent culture where everything is as public as possible and, ideally, written down, then everyone should have more than enough context of what others are working on and why. 
+
+Giving clear ownership over products and projects to individuals or individual teams also lessens the risk of overlap.
+
+We’ve got a small number of product managers who help small teams prioritize, create and update dashboards, conduct competitor analysis, and speak to customers. But we also expect product engineers to be true experts of the product they’re building and understand our customers. Product managers also ensure cross-team alignment, an important task in a completely remote, asynchronous company.
+
+So what about stepping on toes? Well, we [actively encourage that](https://posthog.com/handbook/values). Everyone at PostHog should be raising pull requests in GitHub or proposing changes to stuff that doesn't have anything to do with your small team. Why? Because it shows they really care and they really understand the product.
+
+
+### They have their own mission
+
+Regardless of size or scope, each team has its own mission that feeds into our overall company mission of equipping every developer to build successful products.
+
+On the [marketing team](https://posthog.com/teams/marketing), the mission is to make PostHog a place for product engineers to learn and collaborate. The [infrastructure team’s mission](https://posthog.com/teams/infrastructure) is to make deploying, scaling, and managing PostHog easy, fast, and reliable. But a charter is more than a mission. Small team charters also need to include:
+
+* Long-term goals
+* A description of what the team does
+* A target customer
+* Details of who’s on the team
+* Key metrics
+
+All of this information is in our [handbook](https://posthog.com/handbook) and updated when changes are made and confirmed each quarter. This keeps everyone on the same page and makes it easy for anyone within the company to quickly find out what other teams are working on and why.
+
+### They’re flexible
+
+We’d rather hire new people than keep moving people around to fill gaps. That said, we’re happy for people to move between teams when needed, ideally no more often than every three to nine months. There are two scenarios that could trigger a move:
```

Comment:
3 to 9 months seems like a super wide window. Where is this from?

Reply:
> We try to keep moves infrequent and when needed. We anticipate moving people roughly every 3-9 months. We'd rather hire new people than create gaps by shifting people around.

https://posthog.com/handbook/company/small-teams